test-cases/artifacts/amazon2.yaml

```diff
@@ -0,0 +1,20 @@
+ami_db_scylla_user: 'ec2-user'
+ami_id_db_scylla: 'ami-099a8245f5daa82bf'
+append_scylla_setup_args: '--no-cpuscaling-setup'
```

Comment:
is this official for Amazon Linux?

Reply:
there's no yet official instructions for installation on  Amazon Linux

Reply:
in https://github.com/scylladb/scylla/pull/5540, seem like @syuu1228 doesn't use this option, and can install scylla just fine... (maybe other option for the AMI also disable the same thing)

Reply:
is this temporary, until it will be fixed?

Reply:
If this is an issue need to open one in relevant repo

Reply:
Filed https://github.com/scylladb/scylla/issues/5977